src/gg.regression.unity.bots/Editor/Scripts/Startup/RegressionPackagePopup.cs

```diff
@@ -375,16 +241,16 @@ private void RenderLoginButtonOption()
         GUI.Box(demoInfoBoxRect, "");
         GUI.color = prevColor;
 
-        // Draw the login title
-        GUI.Label(new Rect(20, 450, 580, 20), "Login", titleStyle);
+        // Draw the title
+        GUI.Label(new Rect(20, 450, 580, 20), "API Key", titleStyle);
 
-        // Draw the description for logging in
-        GUI.Label(new Rect(20, 480, 565, 20), "Login to Regression Games to utilize our features.", descriptionStyle);
+        // Draw the description
+        GUI.Label(new Rect(20, 480, 565, 20), "Set an API key to utilize our features.", descriptionStyle);
```

Comment:
let's use the word `setup` instead of `set` both here and on the button

Reply:
Done 👍